### PR TITLE
[arci-ros] Create test for new_joint_position_limiter

### PR DESCRIPTION
**Pull request by chatblanc-ciel**
-  for `fn new_joint_position_limiter`

### CODE REVIEW

File: arci-ros/src/ros_control_client.rs [hunk 1]
```diff
@@ -484,3 +484,47 @@ impl SetCompleteCondition for RosControlClient {
...
+
+        let result = new_joint_position_limiter(client, Some(limits.clone()), None);
+        assert!(result.is_ok());
+        let result = result.unwrap();
+
+        let client = DummyJointTrajectoryClient::new(vec![
+            String::from("a".to_owned()),
+            String::from("b".to_owned()),
+        ]);
+        let correct = JointPositionLimiter::new(client, limits);
```
Comment (chatblanc-ciel) [inline arci-ros/src/ros_control_client.rs:513]: How check private field?

**Review by chatblanc-ciel — COMMENTED** [review]
Help me.

**Comment by chatblanc-ciel** [comment]
Clippy of `GitHub Action` says `"".to_owned()` is error?
I decided these are changed to `String::from()`.

**Comment by OTL** [comment]
> Clippy of `GitHub Action` says `"".to_owned()` is error?
> I decided these are changed to `String::from()`.

Please add the URL when you want to mention about the result of github actions.
I think you're talking about below.
https://github.com/openrr/openrr/runs/3258249000?check_suite_focus=true

It says that do not convert String into String. It does not say "do not use to_owned()".
Please revert `String::from` part.

**Comment by chatblanc-ciel** [comment]
> > Clippy of `GitHub Action` says `"".to_owned()` is error?
> > I decided these are changed to `String::from()`.
> 
> Please add the URL when you want to mention about the result of github actions.
> I think you're talking about below.
> https://github.com/openrr/openrr/runs/3258249000?check_suite_focus=true
> 
> It says that do not convert String into String. It does not say "do not use to_owned()".
> Please revert `String::from` part.

Yes, I get it.　Thanks.

**Review by OTL — APPROVED** [review]

**Pull request merged**